### PR TITLE
Set up PhpStan and Psalm

### PR DESCRIPTION
**Pull request by driehle**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parameters:
+    level: 5
+    paths:
+        - src
+        - test
+includes:
+	- vendor/jangregor/phpstan-prophecy/extension.neon
```
Comment (greg0ire) [inline phpstan.neon]: Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.

Comment (driehle) [reply phpstan.neon]: New line added

**Review by TomHAnderson — APPROVED** [review]
phpstan, psalm, phpunit all running as expected

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @driehle !

**Pull request merged**